CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.12.1
+
+### Changed
+- [[#215](https://github.com/rust-vmm/vmm-sys-util/pull/215)]: Make
+  `as_mut_fam_struct()` public and unsafe to let users modify fields of the
+  header other than the length.
```

Comment:
Shouldn't this have been a v0.13.0 since an access was marked unsafe?

Reply:
The function was made private in 0.12.0, so its essentially just adding a new function.

Reply:
IIUC in v0.12.0 `as_mut_fam_struct()` was marked private by mistake. Some users (e.g. vhost crate) used that API from previous versions, so IMHO v0.12.1 should be fine.

Maybe we should yank v0.12.0 since we did not want to make `as_mut_fam_struct()` not accessible.

Reply:
Ah. Sorry, missed that the removal was done in 0.12.0. Makes sense! I do not think a yank would have been required in that case.

Reply:
> I do not think a yank would have been required in that case.

@Ablu why not?
For example, `vhost` crate broke after updating from v0.11.0 to v0.12.0 because it used `as_mut_fam_struct()`.

Reply:
> For example, `vhost` crate broke after updating from v0.11.0 to v0.12.0 because it used `as_mut_fam_struct()`.

Yes, but that is "fine" as it was a major release which is allowed to break stuff. So if someone would have updated to `0.12.0` (so someone who does not use `as_mut_fam_struct()`), they would have been fine and would not necessarily need to be forced to `0.12.1`. Of course it is unlikely that this happened :)

Reply:
Okay, now I see your point ;-)